server/config/tenant/config.json

```diff
@@ -0,0 +1,16 @@
+{
+    "cs": {
+      "displayName": "Correspondence System",
+      "bulkCreate": true,
+      "deadlines": true,
+      "autoCreateAndAllocate": false,
```

Comment:
Who is this true for?

Reply:
No-one by the looks of it. I'll have a look to see if it's used in the codebase tomorrow.